macroslib/src/python/mod.rs

```diff
@@ -0,0 +1,1219 @@
+use crate::typemap::ast;
+use crate::typemap::ty::RustType;
+use crate::{
+    error::Result,
+    source_registry::SourceId,
+    typemap::{ast::GenericTypeConv, TypeConvCode},
+    types::{
+        ForeignEnumInfo, ForeignInterface, ForeignerClassInfo, ForeignerMethod, ItemToExpand,
+        MethodVariant, SelfTypeVariant,
+    },
+    DiagnosticError, LanguageGenerator, PythonConfig, SourceCode, TypeMap,
+};
+use heck::SnakeCase;
```

Comment:
not used?

Reply:
It is an extension trait. Its method `to_snake_case` is used.